sample-app/src/components/InputDropdown.tsx

```diff
@@ -128,52 +169,40 @@ export default function InputDropdown({
           placeholder={placeholder}
           onChange={evt => {
             const value = evt.target.value;
-            dispatch({ type: 'ShowOptions' });
+            dispatch({ type: 'ShowSections' });
             setLatestUserInput(value);
-            updateInputValue(value);
-            updateDropdown();
+            onInputChange(value);
```

Comment:
I feel like `setInputValue` is a more accurate name. Do you think that works better than `onInputChange`?

Reply:
I think the idea was that this could be used for more than just setting the input. The user can do whatever they want when the input changes, not just set the value

Reply:
got it, we can stick with `onInputChange` then